.github/workflows/build.yaml

```diff
@@ -12,12 +12,11 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: A workaround to increase available disk size
-        run: |
-          sudo rm -rf /usr/share/dotnet
-          sudo rm -rf /opt/ghc
-          sudo rm -rf "/usr/local/share/boost"
-          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+
+      - name: Free disk space
```

Comment:
Did we run into disk size issues , again?

Reply:
Yes, we had. 
We got the HTTPError: 400, just like the first time we got it.